iceoryx_ros2_bridge/src/generic_subscription.cpp

```diff
@@ -88,6 +89,14 @@ void GenericSubscription::return_serialized_message(
   message.reset();
 }
 
+void GenericSubscription::handle_serialized_message(
+  const std::shared_ptr<rclcpp::SerializedMessage> & serialized_message,
+  const rclcpp::MessageInfo & message_info)
+{
+  (void) serialized_message;
+  (void) message_info;
+}
```

Comment:
Didn't we have a way to handle serialized messages?

Reply:
I have not tested the iceoryx_ros2_bridge yet (see [table](https://github.com/ros2/rmw_iceoryx#limitations) in main readme). This change was need as starting with galactic `rclcpp::SubscriptionBase` has a pure virtual method `handle_serialized_message()`. 

@jeising What was your main use-case for the iceoryx_ros2_bridge? Interoperability with other rmw implementations?

Reply:
This is a rename/restructuring not administrated by me. I understand, that ros2 bridge simply holds the functions to connect `ros2 api` via `rmh.h` to `iceoryx`. I thought `handle serialized message` allows us to avoid copying data in the `rmw_iceoryx` layer? There is a good chance my assumptions are mistaken.

Reply:
In fact this class could be completely deleted here. We have that functionality covered upstream in galactic: https://github.com/ros2/rclcpp/blob/galactic/rclcpp/include/rclcpp/generic_subscription.hpp

Reply:
Added #56 to discuss the future of the ros bridge.